src/Segments/VCS.hs

```diff
@@ -3,6 +3,7 @@
 module Segments.VCS where
 
 import Control.Monad (liftM2)
+import Control.Monad.Trans.Maybe
```

Comment:
Please give an explicit imports list here, rather than importing the entire package.